unit/models/application.py

```diff
@@ -195,3 +195,27 @@ def __init__(self, application_id: str, document_id: str, file: IO, file_type: F
         self.file_type = file_type
         self.is_back_side = is_back_side
 
+
+class ListApplicationParams(UnitParams):
```

Comment:
Note the new `UnitParams` parent class. I thought it may be useful to define how these parameter classes convert to dictionaries, so made this new class.